app/helpers/movies_helper.rb

```diff
@@ -34,12 +34,12 @@ def link_to_movie(movie)
   end
 
   def trailer_placeholder_text(movie)
-    change_or_add = movie.trailer.present? ? 'change the' : 'add a'
-    "To #{change_or_add} trailer, enter the YouTube URL"
+    change_or_add = movie.trailer.present? ? 'Change' : 'Add'
+    "Enter YouTube URL to #{change_or_add} Trailer"
   end
 
   def trailer_button_text(movie)
-    movie.trailer.present? ? 'Change trailer' : 'Add a trailer'
+    movie.trailer.present? ? 'Change' : '+ Add'
```

Comment:
@mikevallano and I made this language more concise and now it all fits on one line.